static/js/modals/ImageSelect.tsx

```diff
@@ -69,16 +69,8 @@ const ImageSelect: FC<Props> = ({ onClose, onSelect }) => {
     queryKey: [queryKeys.images, canonicalServer],
     queryFn: () => loadImages(canonicalJson, canonicalServer),
   });
-
-  if (isLciLoading) {
-    return <Loader text="Loading Linux Containers images..." />;
-  }
-
-  if (isCiLoading) {
-    return <Loader text="Loading Canonical images..." />;
-  }
-
-  const images = canonicalImages.concat(linuxContainerImages);
+  const isLoading = isCiLoading || isLciLoading;
```

Comment:
@lorumic made a little adjustment here, to open the modal immediately and show the loading spinner in the table.